books/SPARK_for_the_MISRA_C_Developer/chapters/03_syntactic_guarantees.rst

```diff
@@ -1,4 +1,4 @@
-:code-config:`run_button=False;prove_button=False;accumulate_code=False`
+:code-config:`run_button=True;prove_button=False;accumulate_code=False`
```

Comment:
this does not work, it will add "Run" buttons on the "package sign_domain" and "function absval" snippets, which don't have a main.

Reply:
yes, and in that case it will just compile the code no? I'd like these snippets to be compiled instead of analyzed, so a run button is all I need?